src/App.css

```diff
@@ -0,0 +1,437 @@
+  color: var(--light-pink-text-color);
+  -webkit-text-stroke: 2px #fff;
+}
+
+.aside-span-effect:before {
+  content: attr(data-fill);
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  color: #fff;
+  white-space: nowrap;
+  transition: 0.8s;
+  overflow: hidden;
+  clip-path: polygon(0 0, 0 1%, 100% 1%, 100% 0);
+}
+
+.aside-span-effect:hover:before {
+  height: 100%;
+  clip-path: polygon(0 0, 0 100%, 100% 100%, 100% 0);
+}
+
+.footer {
+  background: var(--footer-background-color);
+  grid-row: 3;
+  grid-column: 1/4;
+  display: flex;
+  flex-direction: column;
+  color: var(--background-color);
+  align-items: center;
+  justify-content: center;
+  background-color: #262626;
+}
+
+.img24 {
+  filter: grayscale(80%);
+  opacity: 0.8;
+}
+
+.img26 {
+  filter: grayscale(80%);
+  opacity: 0.8;
+}
+
+.alice-carousel__dots {
+  display: none;
+}
+
+p.alice-carousel__prev-btn-item,
+p.alice-carousel__next-btn-item {
+  display: none;
+}
+
+div.alice-carousel__prev-btn,
+div.alice-carousel__next-btn {
+  display: none;
+}
+
+.alice-carousel ul li img {
+  display: flex;
+  height: 350px;
+  width: 50%;
+  object-fit: cover;
+}
+
+.footer-text {
+  margin-top: 3px;
+  font-size: 10px !important;
+}
+
+.footer-contacts {
+  margin-top: 8px;
+}
+
+.footer-contacts img {
+  margin-right: 5px;
+}
+
+.footer-signature {
+  margin: 5px;
+  font-size: 10px !important;
+  font-family: "Lucida Handwriting";
+}
+
+.header {
+  color: white;
+}
+
+.slidertext {
+  width: 50%;
+  height: 100px;
+}
+
+blockquote {
+  text-align: end;
+  font-family: "Gilroy", sans-serif;
+}
+
+.quote1 {
+  font-family: inherit;
+  font-size: 20px;
+  width: 380px;
+}
+
+.quote2 {
+  font-family: inherit;
+  font-size: 20px;
+  width: 420px;
+}
+
+.quote3 {
+  font-family: inherit;
+  font-size: 20px;
+  width: 660px;
+}
+
+.quote4,
+.quote5 {
+  font-family: inherit;
+  font-size: 20px;
+  width: 580px;
+}
+
+.quote5 {
+  width: 470px;
+}
+
+.author {
+  font-family: "Gilroy", sans-serif;
+  font-size: 18px;
+  border-right: 4px solid var(--light-pink-text-color);
+  padding-right: 10px;
+}
+
+.sidekick:before,
+.sidekick:after {
+  font-family: Calibri;
+  color: var(--light-pink-text-color);
+  font-size: 22px;
+  margin-right: 5px;
+}
+.sidekick:before {
+  content: "\201c";
+  margin-right: 5px;
+  margin-left: 20px;
+}
+.sidekick:after {
+  content: "\201c";
+  margin-left: 3px;
+}
+
+.alice-carousel__wrapper {
+  width: 800px !important;
+}
+
+.section-carousel {
+  height: 100%;
+}
+
+.section-carousel .alice-carousel__wrapper {
+  display: flex;
+  align-content: center;
+  width: 100%;
+  height: 100%;
+  width: 400px !important;
+  margin-right: 0.5em;
+}
+
+.section-carousel .alice-carousel ul li img {
+  height: 100%;
+  width: 100%;
+
+  /* height:490px; */
+}
+
+.sliderimg {
+  object-fit: cover;
+  -webkit-filter: grayscale(100%);
+  filter: grayscale(100%);
+  border: 2px solid #000;
+  box-shadow: 2px 2px 8px 12px rgba(0, 0, 0, 0.8);
```

Comment:
> I'd like to add a blurred effect on all the borders of the images. If you pay attention to the images, you'll see that the blurred effect only applies to the right border of images.

For this one approach is to use [`box-shadow: inset`](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow), where the shadow is applied to inside the image instead of the outside.

Watch the video  I sent you over discord to understand why the box-shadow does not work directly in a `<img>` element. Instead, we need to apply it to another element. I suggest applied it to the closer element - the parent slide item, inside a pseudo selector.

```css
.alice-carousel__stage-item::after {
  content: "";
  position: absolute;
  display: block;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  box-shadow: inset 0 0 10px 12px rgba(0, 0, 0, 0.8);
}

```